client/src/plus/deep-dives/sample-articles.tsx

```diff
@@ -395,8 +395,8 @@ export function YourBrowserSupportToolkit() {
         <h2>Modern CSS in the real world : Part two</h2>
       </header>
       <p className="article-lead">
-        Viverra neque eget tellus, gravida pellentesque non. Purus, venenatis
-        sit ac nam nec. At vitae facilisi in mi amet aliquam.
+        In this article discover the resources available, to help you develop a
+        site that will perform well across browsers and devices.
```

Comment:
This is incorrect now. Merge in the latest `main` and resolve the conflict.

Reply:
Done.

Reply:
I don't get why but she wants the sentences to now end with a period. 
See https://github.com/mdn/yari/pull/4291

Reply:
It does have a period though? Or are you saying there should not be a period?

Reply:
> It does have a period though? Or are you saying there should not be a period?

Not have a period. Don't ask me why. 
See conversation in https://github.com/mdn/yari/pull/4291